CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased changes
+- Removed the configuration option 'no_rebroadcast_consensus_validation' as it is never used.
```

Comment:
```suggestion
- Removed the configuration option 'no_rebroadcast_consensus_validation'.
```

"Never used" is not really accurate. It is an option and somebody could have used it.